modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/db/Statement.scala

```diff
@@ -47,6 +47,7 @@ object Statement {
   case object Commit extends Statement
   case object Abort extends Statement
   case object Select1 extends Statement
+  case object ReadyCheck extends Statement
```

Comment:
Cannot we use `Select1` for it? I just don't want to grow the ADT too much without necessity.

Reply:
`ReadyCheck` statement for Snowflake is different than others. It is [ALTER WAREHOUSE ... RESUME IF SUSPENDED](https://github.com/snowplow/snowplow-rdb-loader/blob/feature/databricks-loader/modules/snowflake-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/snowflake/Snowflake.scala#L85).  I've removed `Transaction.resume`. It was only for Snowflake. Unifying them like this seems better to me.